### PR TITLE
Implement FlowDirection property in ViewHandlers

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change ###

- Implement `FlowDirection ` property in ViewHandlers.
- Don't force the platform to LTR. If the FlowDirection matches the windows FlowDirection then don't set anything on the native platform

### Platforms Affected ### 

- Core
- iOS
- macOS
- Android
- Windows

### PR Checklist ###

- [x] Targets the correct branch 
- [ ] Tests are passing (or failures are unrelated)
- [x] Targets a single property for a single control (or intertwined few properties)
- [x] Adds the property to the appropriate interface
- [x] Avoids any changes not essential to the handler property
- [x] Adds the mapping to the PropertyMapper in the handler
- [x] Adds the mapping method to the Android, iOS, and Standard aspects of the handler
- [x] Implements the actual property updates (usually in extension methods in the Platform section of Core)
- [x] Tags ported renderer methods with [PortHandler]
- [x] Adds an example of the property to the sample project (MainPage)
- [x] Adds the property to the stub class
- [x] Implements basic property tests in DeviceTests

### CODE REVIEW

File: src/Core/src/Platform/Android/ViewExtensions.cs [hunk 1]
```diff
@@ -48,6 +50,24 @@ public static void UpdateBackground(this AView nativeView, IView view, Drawable?
...
 		}
 
+		public static void UpdateFlowDirection(this AView nativeView, IView view)
+		{
+			if (view.FlowDirection.IsRightToLeft())
+			{
+				nativeView.LayoutDirection = ALayoutDirection.Rtl;
+
+				if (nativeView is TextView textView)
+					textView.TextDirection = ATextDirection.Rtl;
```
Comment (AmrAlSayed0) [inline src/Core/src/Platform/Android/ViewExtensions.cs:60]: Isn't HorizontalTextAlignment different from FlowDirection? You could have a Label with RTL FlowDirection and Arabic text for example but with HorizontalTextAlignment set to "End".

Comment (hartez) [reply src/Core/src/Platform/Android/ViewExtensions.cs:60]: Yes, text alignment is different than flow direction. But this method doesn't affect text alignment.

Comment (AmrAlSayed0) [reply src/Core/src/Platform/Android/ViewExtensions.cs:60]: Oh, sorry, I misunderstood 😅

**Review by rachelkang — CHANGES_REQUESTED** [review]
Text is not getting rendered as RTL

Screenshot_1622233079

**Review by rmarinho — CHANGES_REQUESTED** [review]
I found a couple of issus, some controls don't support RTL. 
- Starting app with Hebrew (RTL language) doesn't show the app in RTL on iOS
- Forcing RTL on my android Device and starting the app doesnt show the sample app with RTL on

**Review by rmarinho — APPROVED** [review]

**Review by rachelkang — APPROVED** [review]

**Pull request merged**